examples/gan/pix2pix_tf_keras/export.py

```diff
@@ -0,0 +1,73 @@
+"""
+Shows an example
+"""
+
+import argparse
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from determined import keras
+from determined.experimental import client
+
+from data import download, load_dataset
+
+
+def generate_and_plot_images(generator: tf.keras.Sequential) -> None:
+    path = download("http://efrosgans.eecs.berkeley.edu/pix2pix/datasets/", "facades")
```

Comment:
same here

Reply:
Since this code is meant to be run after a Determined experiment, your point is more important that it is with `data`'s `main`. If we're going to make your suggested change to either, we might as well do it to both, so I'll get on it! :-)